Doc/Learning/learning.mixer

```diff
@@ -87,21 +87,11 @@ Of this the section 'Graph Theory using the GRAPE package' was contributed by
   <li> 
     <mixer person="David Joyner" data="name_link"/>
     is collecting a list of frequently asked questions about 
-    <a href="https://yetanothermathblog.com/2017/08/09/how-do-i-construct-in-gap/">
+    <a href="https://yetanothermathblog.com/the-toric-package-for-gap/how-do-i-construct-in-gap/">
     Constructions</a> of various mathematical objects in GAP
     with fully worked out GAP code answering them. This 
     collection is specially recommended for newcomers to the system . 
   <br />&nbsp;  
-  </li>
-  <li>
```

Comment:
@DominikBernhardt please restore this block - thanks to @hulpke, the material  has been recovered